group_vars/all/nginx.yml

```diff
@@ -0,0 +1,71 @@
+nginx_remove_default_vhost: true
+nginx_user: www-data
+nginx_vhosts:
```

Comment:
Why? It's the default in geerlingguy's nginx role (https://github.com/geerlingguy/ansible-role-nginx)

Reply:
I also think the way with templates would be cleaner. then this could look something like this:
```yaml
nginx_vhosts:
  - server_name: "chaos.jetzt"
    template: "{{ playbook_dir }}/../templates/chaos.jetzt.vhost.j2"
```

Reply:
> Why? It's the default in geerlingguy's nginx role (https://github.com/geerlingguy/ansible-role-nginx)

It seems not so clean. In the end, we have a config file that's being generated by dumping a good portion of the final config file in a yaml block.